lib/string_reverse.rb

```diff
@@ -1,4 +1,17 @@
 # A method to reverse a string in place.
 def string_reverse(my_string)
-  my_string << "not implemented"
+   if my_string == ""
+     return my_string
+   elsif my_string.nil?
+     return nil

```

Comment:
Technically, since you're reversing the string inplace, you don't need to return anything in particular and lines 4 and 6 could simply be `return`